game/script-events.rpy

```diff
@@ -3444,10 +3467,399 @@ label holiday_easter:
+    $ jnPause(1)
+
+    # Have to proc manually here due to fake reveal for the prank
+    $ renpy.play(filename="mod_assets/bgm/vacation.ogg", channel="music")
+    $ renpy.show_screen("hkb_overlay")
+    $ jn_globals.force_quit_enabled = True
+
+    $ player_upper = player.upper()
+    n 4fchbg "HAPPY HALLOWEEN,{w=0.75}{nw}" 
+    extend 4fchbs " [player_upper]!"
+    n 4fchsm "..."
+    n 4fsqsm "..."
+    n 2fsqss "Well?{w=0.75}{nw}"
+    extend 2fcsgs " Don't just sit there!{w=0.75}{nw}"
+    extend 4fchgn " Say something already!"
+    n 3fnmbs "Did I get you?{w=0.2} Did I get you?{w=0.75}{nw}"
+    extend 3fcsbs " Don't lie!{w=1}{nw}"
+    extend 7fsqbg " I {i}totally{/i} got you this time!"
+    n 3ccsbg "Jeez...{w=1}{nw}"
+    extend 7fcsbs " I {i}knew{/i} this dumb old book would come in handy someday!\n{w=0.75}{nw}"
+    extend 7fsqss "Not bad for a bunch of reused scripts,{w=0.2} huh?"
+    n 3unmbg "And hey -{w=0.5}{nw}"
+    extend 3tlrbg " I didn't even blow the classroom up or anything.{w=0.75}{nw}"
+    extend 4fchbg " I'd call that a complete success!"
+    n 2fchsm "Ahaha."
+    n 2cllss "Well,{w=0.2} anyway.{w=0.75}{nw}"
+    extend 2ccsbg " I think I'm about done screwing around now."
+    n 4ullaj "So -{w=0.5}{nw}"
+    extend 4csqbg " you like what I've done with the place?{w=0.75}{nw}"
+    extend 3ccspo " You better appreciate all the effort I put into all this,{w=0.2} [player]."
+    n 3fnmbg "Yep!{w=0.75}{nw}"
+    extend 7fcsbg " As you can {i}clearly{/i} see -{w=0.5}{nw}"
+    extend 6fchbg " I pulled out {i}all{/i} the stops this time!"
+    n 7clrbg "Awesome hand-crafted decorations?{w=0.75}{nw}"
+    extend 7ccsbg " Check!{w=1}{nw}"
+    extend 6ckrsm " Fabulously made cosplay?{w=0.75}{nw}"
+    extend 6fcsbs " Check again!"
+    n 3fnmbg "Getting easily the best jump of the year on [player]?"
+    n 7fsgsm "..."
+    n 7fcssm "Ehehe.{w=0.75}{nw}"
+    extend 3fchgnelg " You bet!"
+
+    if get_topic("talk_thoughts_on_horror").shown_count > 0:
+        n 1cllbg "Man..."
+        n 2unmaj "You know,{w=0.2} I'm pretty sure I mentioned it before at some point.{w=0.75}{nw}"
+        extend 5csrbosbl " About how I don't really like horror and all."
+        n 4ccsfl "But let's be real here,{w=0.2} [player]."
+
+    else:
+        n 1cllbg "Man...{w=1}{nw}"
+        extend 2tlraj " I know I told you I wasn't the biggest fan of horror already,{w=0.75}{nw}" 
+        extend 4ccsfl " but let's be real."
+
+    n 4tnmaj "When it really comes to bang for your buck?{w=0.75}{nw}"
+    extend 7ccssmesm " You just can't beat a good old Halloween."
+    n 3unmfl "No,{w=0.2} really!{w=0.75}{nw}"
+    extend 3fcsbg " Think about it!"
+    n 4fsqss "Not {i}only{/i} is it an excuse to break out my sewing kit and blow {i}everyone{/i} away with my needlework..."
+    n 1ccsss "But come on.{w=0.75}{nw}"
+    extend 2fchgn " What {i}other{/i} holidays give you the excuse to pig out on as much free candy as you can swipe,{w=0.2} huh?"
```

Comment:
Nice reference to the official halloween art!

Reply:
Accidental reference, but thank you